packages/core/src/utils/CoreUtil.ts

```diff
@@ -88,6 +90,10 @@ export const CoreUtil = {
     )
   },
 
+  removeWalletConnectDeepLink() {
+    localStorage.removeItem(CoreUtil.WALLETCONNECT_DEEPLINK_CHOICE)
```

Comment:
localStorage wont work in react-native, but i guess this function will be used in web, right?

Reply:
Yeh, this functionality is something you will need to replicate in rn using their storage mechanisms 👍